CMakeLists.txt

```diff
@@ -509,22 +505,20 @@ endif()
 
 # GUI
 if(GUI)
-    qt_add_executable(${GUI_TARGET} MANUAL_FINALIZATION MACOSX_BUNDLE WIN32 ${PROJECT_SOURCES} ${QM_FILES} ${RESOURCES} ${EXTRA_RESOURCES_GUI})
+    qt_add_executable(${GUI_TARGET} MANUAL_FINALIZATION MACOSX_BUNDLE WIN32 ${PROJECT_SOURCES} ${QM_FILES} ${RESOURCES} ${EXTRA_RESOURCES_GUI} icons/icons.qrc)
```

Comment:
Please add icons/icons.qrc in the RESOURCES variable

Reply:
We don't need or want them for the cli target.